src/components/About/About.css

```diff
@@ -0,0 +1,100 @@
+.about {
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+}
+.about__container {
+  display: flex;
+  margin: 80px 24px;
+  max-width: 1280px;
+  box-sizing: border-box;
+  width: 100%;
+}
+.about__author--profile {
+  width: 464px;
+  height: 464px;
+  border-radius: 50%;
+  object-fit: cover;
+  padding: 0;
+  margin: 0;
+}
+.about__author--info {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding: 51px 0 0 56px;
+  gap: 24px;
+}
+.about__author--title {
+  font-family: "Roboto Slab";
```

Comment:
It's important to always provide **fallback** and **generic** font name for `font-family`, like that: 

```css

font-family: "Roboto Slab ", Arial, sans-serif;


```

https://developer.mozilla.org/en/docs/Web/CSS/font-family

Font names are just for an example, you have your own font names and need to set fallback fonts based on it. For example, `Noto Serif` and `Inter` belong to different types, and they need different fallback fonts.
https://www.w3schools.com/css/css_font.asp 

Please, make this correction for **all the places** where you use `font-family`